pom.xml

```diff
@@ -376,7 +375,7 @@
                         <exclude>**/src/test/resources/**</exclude>
                         <exclude>**/target/**</exclude>
                     </excludes>
-                    <lineEnding>LF</lineEnding>
+                    <lineEnding>AUTO</lineEnding>
```

Comment:
Nope not true.  Code is checked out as auto so it matches box.  See git attributes plus the fact so far both linux and my local windows passed.


Sent by Outlook for Android





On Wed, Oct 28, 2015 at 9:13 PM -0700, "Marvin Froeder" <notifications@github.com> wrote:
> @@ -376,7 +375,7 @@
>                          <exclude>**/src/test/resources/**</exclude>
>                          <exclude>**/target/**</exclude>
>                      </excludes>
> -                    <lineEnding>LF</lineEnding>
> +                    <lineEnding>AUTO</lineEnding>

this ain't a good idea

AUTO for windows is different from mac that is in time different from linux.

So our build become non-deterministic

---
Reply to this email directly or view it on GitHub:
https://github.com/revelc/formatter-maven-plugin/pull/80/files#r43347516